pkg/controller/container-runtime-config/helpers.go

```diff
@@ -123,6 +123,7 @@ func createNewRegistriesConfigIgnition(registriesTOMLConfig []byte) ignv2_2types
 		}
 		tempIgnConfig.Storage.Files = append(tempIgnConfig.Storage.Files, regTempFile)
 	}
+	tempIgnConfig.Ignition.Version = "2.2.0"
```

Comment:
Is there any way to avoid hardconding the version here? Floating strings are really bad generally (not your fault, just want to avoid them as much as possible)

Reply:
I think we probably want a common `NewIgnitionConfig()` helper somewhere, i.e.:
`tempIgnConfig := ignitionhelper.NewIgnitionConfig()`
and have that set the version.